spanner/spansql/parser.go

```diff
@@ -982,7 +982,7 @@ func (p *parser) parseDDLStmt() (DDLStmt, *parseError) {
 
 	/*
 		statement:
-			{ create_database | create_table | create_index | alter_table | drop_table | drop_index | create_change_stream | alter_change_stream | drop_change_stream }
+			{ create_database | create_table | create_index | alter_table | drop_table | rename_table | drop_index | create_change_st     ream | alter_change_stream | drop_change_stream }
```

Comment:
```suggestion
			{ create_database | create_table | create_index | alter_table | drop_table | rename_table | drop_index | create_change_stream | alter_change_stream | drop_change_stream }
```

Reply:
Thank you! Did not realize the mistake. First time on github or open-source.